chains/solana/contracts/programs/ccip-common/src/v1.rs

```diff
@@ -272,29 +275,15 @@ fn load_v1_token_admin_registry(
     );
 
     require!(
-        TokenAdminRegistry::DISCRIMINATOR == data[..8],
+        TokenAdminRegistry::DISCRIMINATOR == data[..ANCHOR_DISCRIMINATOR_SIZE],
```

Comment:
This is just a nit, and one that there's no need to act on now as it's pervasive on the codebase, but for situations like this in the future I think it's a bit cleaner to use [split_at](https://doc.rust-lang.org/std/primitive.slice.html#method.split_at) rather than using slice ranges.